docs/using.md

```diff
@@ -5,86 +5,56 @@ overview of different components and services.
 
 <!-- MarkdownTOC -->
 
-- [Managing your OpenShift instance](#managing-your-openshift-instance)
-  - [Starting OpenShift](#starting-openshift)
-  - [Stopping OpenShift](#stopping-openshift)
-  - [Deleting OpenShift](#deleting-openshift)
-  - [Updating OpenShift configuration](#updating-openshift-configuration)
+- [Managing Minishift](#managing-minishift)
+  - [Minishift start](#minishift start)
+  - [Minishift stop](#minishift-stop)
+  - [Minishift delete](#minishift-delete)
 - [Minishift runtime options](#minishift-runtime-options)
   - [Flags](#flags)
   - [Environment variables](#environment-variables)
   - [Persistent configuration](#persistent-configuration)
     - [Setting persistent configuration values](#setting-persistent-configuration-values)
     - [Unsetting persistent configuration values](#unsetting-persistent-configuration-values)
+- [HTTP/HTTPS Proxies](#httphttps-proxies)
+- [Mounted host folders](#mounted-host-folders)
+  - [Mounting custom shared folders](#mounting-custom-shared-folders)
+- [Networking](#networking)
 - [Interacting with OpenShift](#interacting-with-openshift)
   - [OpenShift client binary \(oc\)](#openshift-client-binary-oc)
   - [Login](#login)
   - [Console](#console)
   - [Services](#services)
-- [HTTP/HTTPS Proxies](#httphttps-proxies)
-- [Mounted host folders](#mounted-host-folders)
-  - [Mounting custom shared folders](#mounting-custom-shared-folders)
-- [Networking](#networking)
-- [Persistent volumes](#persistent-volumes)
-- [Private container registries](#private-container-registries)
 
 <!-- /MarkdownTOC -->
 
-<a name="managing-your-openshift-instance"></a>
-## Managing your OpenShift instance
+<a name="managing-minishift"></a>
+## Managing Minishift
+When you use Minishift, you actually interact with:
+- the virtual machine(VM) created by Minishift and
+- the OpenShift instance provisioned by the VM.
```

Comment:
here a diagram would be awesome. See also #403. I like the one referenced in the issue. Might needs some minor tweaks. Might be outside of the scope of this issue, even though we could add a single png for now.